applications/addons/settings/bootstrap.php

```diff
@@ -0,0 +1,3 @@
+<?php if (!defined('APPLICATION')) exit();
+
+require_once(PATH_APPLICATIONS . DS . 'addons' . DS . 'library' . DS . 'functions.render.php');
```

Comment:
Interesting; I hadn't looked that closely but that is a bit a weird use case. I guess just leave it as-is then since I don't have a better idea.